lib/openstudio-standards/standards/necb/common/btap_data.rb

```diff
@@ -433,6 +433,15 @@ def utility(model)
           AND ColumnName='#{ep_fuel} Annual Value'
           AND Units='GJ'"
         fuel_consumption_gj = model.sqlFile.get.execAndReturnFirstDouble(sql_command).is_initialized ? model.sqlFile.get.execAndReturnFirstDouble(sql_command).get : 0.0
+      elsif neb_fuel == 'Natural Gas'
```

Comment:
With the above fixed NEB-> E+ mapping fixed... you don't have to repeat yourself with the elsif statement. Less code == less errors.  You do not have to change this if you do not want to.